pallets/phala/src/mining.rs

```diff
@@ -447,7 +447,7 @@ pub mod pallet {
 			let who = ensure_signed(origin)?;
 			let pubkey = Self::ensure_miner_bound(&miner)?;
 			let worker =
-				registry::Workers::<T>::get(&pubkey).ok_or(Error::<T>::WorkerNotRegistered)?;
```

Comment:
Could you skip mining related files? There's an ongoing big PR on the way. It will create a lot of conflicts.

Reply:
Let it go first

Reply:
The StakePool v2 PR will not be merged before Nov. So we can skip mining.rs & stakepool.rs in this PR and merge first

Reply:
OK